apps/web/components/Editor/Editor.vue

```diff
@@ -64,5 +44,26 @@ const {
   formatJson,
   purgeJson,
   clearText,
-} = useJsonEditor(initialJson);
+} = useJsonEditor(JSON.stringify(data, null, 2));
+
+const cleanJsonData = (data: JsonData): JsonData => {
+  if (data._object) {
+    data = data._object;
+  }
+  if (data.data) {
+    data = data.data;
+  }
+  delete data._key;
+  delete data.__v_isRef;
+  return data;
+};
+
+watch(
+  () => data,
+  (updatedData) => {
+    const cleanedData = cleanJsonData(updatedData);
+    jsonText.value = JSON.stringify(cleanedData, null, 2);
```

Comment:
```suggestion
    jsonText.value = JSON.stringify(updateData.value, null, 2);
```
`cleanJsonData` shouldn't be necessary. Same for the corresponding type.